python/pyspark/pandas/tests/test_dataframe.py

```diff
@@ -1900,6 +1900,29 @@ def test_isin(self):
         with self.assertRaisesRegex(TypeError, msg):
             psdf.isin(1)
 
+    def test_mode(self):
+        pdf = pd.DataFrame(
+            [("bird", 2, 2), ("mammal", 4, 0), ("arthropod", 8, 0), ("bird", 2, np.nan)],
+            index=("falcon", "horse", "spider", "ostrich"),
+            columns=("species", "legs", "wings"),
+        )
+        psdf = ps.from_pandas(pdf)
+        self.assert_eq(
+            psdf.mode(),
+            pdf.mode(),
+        )
+        self.assert_eq(
+            psdf.mode(numeric_only=True),
+            pdf.mode(numeric_only=True),
+        )
```

Comment:
Could you add tests with `pdf[[]].mode()`?

Reply:
For sure